teleirc.js

```diff
@@ -110,6 +110,19 @@ tgbot.on('message', function (msg) {
                 let username = msg.new_chat_member.username;
                 let first_name = msg.new_chat_member.first_name;
                 ircbot.say(config.channel, "New user " + first_name + " ( @" + username + ") has joined the Telegram Group!");
+            } else if (msg.sticker != undefined) {
```

Comment:
`!==` should be used here. Triple equals does strict type checking, and is what is used in the other "undefined" checks in the application.